src/site/stages/build/drupal/graphql/GetAllPages.graphql.js

```diff
@@ -52,8 +58,13 @@ module.exports = `
   ${bioPage}
   ${benefitListingPage}
   ${eventListingPage}
+`;
 
-  query GetAllPages($today: String!, $onlyPublishedContent: Boolean!) {
+  const todayQueryVar = useTomeSync ? '' : '$today: String!,';
```

Comment:
Why pull this out into a JS variable?

Reply:
Because in the full graphql build `today` is used, but not in the tome-sync one:

- [used](https://github.com/department-of-veterans-affairs/vets-website/pull/11287/files#diff-79dec7e507c2e256832946af544cfcdfR112)
- [not used](https://github.com/department-of-veterans-affairs/vets-website/pull/11287/files#diff-79dec7e507c2e256832946af544cfcdfR122)